python/tutorials/01-vector-add.py

```diff
@@ -23,6 +23,9 @@
 import triton
 import triton.language as tl
 
+DEVICE = torch.device(triton.runtime.driver.active.get_current_target().backend,
```

Comment:
I think you still need to convert "hip" to "cuda" even when passing it to `torch.device`. It would make sense to add a helper in `triton.runtime`. Something like `get_active_torch_device`.

Reply:
Can I implement `get_active_torch_device` in this PR?

Reply:
yeah that should simplify this a fair bit

Reply:
done